jobrunner/executors/local.py

```diff
@@ -120,6 +120,7 @@ def terminate(self, job):
     def cleanup(self, job):
         cleanup_job(job)
         RESULTS.pop(job.id, None)
+        return JobStatus(ExecutorState.UNKNOWN)
```

Comment:
Is this for consistency, or is it actually useful in some way I've missed?

Not a blocker, just curious.

Reply:
It's what the contract says. And I think that's right for consistency -- all the state transition methods return the new state.

Reply:
(_And_ it's simpler for the decorator if the implementations fulfill the contract rather than having to cope with `None` at that point sometimes.)